vita3k/compat/src/compat.cpp

```diff
@@ -160,12 +166,15 @@ bool update_compat_app_db(GuiState &gui, EmuEnvState &emuenv) {
     const auto app_compat_db_link = "https://github.com/Vita3K/compatibility/releases/download/compat_db/app_compat_db.xml";
 
 #ifdef WIN32
-    const auto download_command = fmt::format("powershell Invoke-WebRequest {} -Outfile {}", app_compat_db_link, compat_db_path.string());
+    const auto download_command = fmt::format(R"(powershell Invoke-WebRequest {} -Timeout 2 -Outfile \"{}\")", app_compat_db_link, compat_db_path.string());
 #else
-    const auto download_command = fmt::format("curl -L {} -o {}", app_compat_db_link, compat_db_path.string());
+    const auto download_command = fmt::format(R"(curl -m 2 -L {} -o "{}")", app_compat_db_link, compat_db_path.string());
```

Comment:
The point of this is to make a network request? There are library to do this consistently (libcurl/cpr) and I think we already have a couple net request libraries. I'll assume this works well for the time being if you don't want to tackle one of these libraries, I don't have the time right now to rework this.